cypress/integration/pages/onDemandRadio/index.js

```diff
@@ -12,11 +12,19 @@ Object.keys(config)
   .forEach(serviceId => {
     const { variant, name: service } = config[serviceId];
     const paths = getPaths(serviceId, pageType);
+
     paths.forEach(currentPath => {
       describe(`${pageType} - ${currentPath}`, () => {
         before(() => {
           Cypress.env('currentPath', currentPath);
-          visitPage(currentPath, pageType);
+          let newPath;
+          if (Cypress.env('APP_ENV') === 'test') {
+            newPath = `${currentPath}?renderer_env=live`;
+          } else {
+            newPath = currentPath;
+          }
+
+          visitPage(newPath, pageType);
```

Comment:
You can technically use the dataEndpointOverride function here (prevents duplication https://github.com/bbc/simorgh/blob/17e23c745f2c0fa6f20f7ab065c79976a9046a17/cypress/support/helpers/onDemandRadioTv.js#L55)